### PR TITLE
Fix use of != on Strings in QueueItem

### PR DESCRIPTION
**Pull request by davidjrichardson**

### CODE REVIEW

File: src/main/java/mcnutty/music/get/ProcessQueue.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //This software is licensed under the GNU GPL v3
-//Written by William Seymour
+//Written by William Seymour and Christopher Leonard
```
Comment (Veltas) [inline src/main/java/mcnutty/music/get/ProcessQueue.java:2]: Hey don't blame me for this @davidjrichardson

**Comment by mcnutty26** [comment]
The only commit that's related to the title is the last one?

**Comment by davidjrichardson** [comment]
The commits needs squishing

**Comment by mcnutty26** [comment]
Please can you make a new request with just the bug fix (no style changes etc.)

**Comment by davidjrichardson** [comment]
If you actually check the code, that's not the case.

**Pull request closed without merging**